### PR TITLE
[BUG] Handle get key operation when no tag has been set

### PR DESCRIPTION
**Pull request by PatStLouis**
Handle cases where a key doesn't have a tags property.

@dbluhm @jamshale small but critical bug introduced by my last PR, I was under the impression every key had tags by default but it doesn't seem like so, this will handle such cases.

### CODE REVIEW

File: aries_cloudagent/wallet/askar.py [hunk 1]
```diff
@@ -192,8 +192,14 @@ async def get_signing_key(self, verkey: str) -> KeyInfo:
         if not key:
             raise WalletNotFoundError("Unknown key: {}".format(verkey))
         metadata = json.loads(key.metadata or "{}")
+
+        # Provisioned keys don't get tags by default
+        try:
+            kid = key.tags["kid"] if "kid" in key.tags else None
```
Comment (jamshale) [inline aries_cloudagent/wallet/askar.py:198]: You can do `key.tags.get('kid')` instead of this. The default if it doesn't exist is `None`.

**Comment by PatStLouis** [comment]
another fix could be to change this line:
https://github.com/OpSecId/aries-cloudagent-python/blob/036f108a5ea4fbc4805b49be6bdc8fd02b1511f6/aries_cloudagent/wallet/askar.py#L100

from:
```python
if kid:
    tags = {"kid": kid}
else:
    tags = None
```

to 

```python
if kid:
    tags = {"kid": kid}
else:
    tags = {}
```

I believe this would be more consistent as a fix since all keys will have a tags object. Any opinion?

File: aries_cloudagent/wallet/askar.py (hunk 1 above)
Comment (PatStLouis) [reply aries_cloudagent/wallet/askar.py]: thanks, I prefer this solution which I've just commited

Comment (PatStLouis) [reply aries_cloudagent/wallet/askar.py]: actually this might not fix it as the problem is that tags doesn't exist. See my second comment.

**Comment by PatStLouis** [comment]
I made a combination of @jamshale suggestion and the latter idea.

**Review by jamshale — APPROVED** [review]

**Pull request merged**